src/frontend/utils/archiveNotesUtils/addToArchives.js

```diff
@@ -0,0 +1,7 @@
+import { addToArchiveService } from "frontend/services";
+
+const addToArchives = async (token, note, notesDispatch) => {
+  await addToArchiveService(token, note);
+  notesDispatch({ type: "ADD_TO_ARCHIVE", payload: note });
+};
+export { addToArchives };
```

Comment:
You could have created one file for archive service and add all the crud apis to it, as "addToArchiveService" doesn't sound like a method name. Just a suggestion.

Reply:
okay. will think on it